apps/cow-tools/src/app/milkman/layout.tsx

```diff
@@ -13,17 +15,22 @@ export const metadata: Metadata = {
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
-    <div className="flex h-full flex-col">
-      <Header
-        linkUrl={"/milkman"}
-        title={APP_DISPLAY_NAME}
-        imageSrc={"/favicon.ico"}
-        endButton={<CustomConnectButton showNetworkSelector={false} />}
-      />
+    <NetworksContextProvider>
+      <div className="flex flex-col h-screen">
+        <Header
+          linkUrl={"/milkman"}
+          imageSrc={"/assets/milkman.svg"}
+          endButton={<CustomConnectButton showNetworkSelector={false} />}
+        />
 
-      <div className="sm:flex flex-1 gap-x-8 px-4 pt-4 text-white">
-        {children}
+        <div className="flex flex-1 overflow-auto p-4 sm:flex-row sm:gap-x-8 text-white">
+          {children}
+        </div>
+        <Footer
+          githubLink="https://github.com/bleu-fi"
+          discordLink="https://discord.gg/Z78vQvmHN2"
+        />
```

Comment:
All right, you moved the footer from another layout to here, right? Sounds good for me

Reply:
https://github.com/bleu-fi/balancer-tools/pull/449#discussion_r1376076336